processor/src/main/kotlin/com/syouth/kmapper/processor/base/Utils.kt

```diff
@@ -51,7 +51,8 @@ internal fun KSFunctionDeclaration.getMappingInformation(): MapperInformation {
 
 internal fun KSClassDeclaration.buildMappingTable(from: KSClassDeclaration): List<MappingProperties> {
     val result = mutableListOf<MappingProperties>()
-    val toProperties = primaryConstructor?.parameters ?: throw IllegalStateException("Data class should have primary constructor")
+    val toProperties = primaryConstructor?.parameters
+        ?: throw IllegalStateException("Data class $classKind ${from.simpleName.asString()} should have primary constructor")
```

Comment:
This is not only for data classes only anymore. Please use "Class $classKind ${from.simpleName.asString()} should have primary constructor"